Source/FakeItEasy/IArgumentConstraintManager.cs

```diff
@@ -1,7 +1,7 @@
 ﻿namespace FakeItEasy
 {
+    using Configuration;

```

Comment:
Introducing this `using` before the "System" ones has created a warning about using order:

http://teamcity.codebetter.com/viewLog.html?buildId=208583&buildTypeId=bt929&tab=buildLog#_state=50,122&focus=125

(you can see it when you build in the IDE or command line as well).

If you're not sick of us nitpicking yet, would you consider moving this using below line 5?
If you are, I can submit a superseding PR that would do it. You'd still get credit for your commit.